.travis.yml

```diff
@@ -14,7 +14,6 @@ install:
   - sudo chmod +x /usr/local/bin/src
   - src toolchain add sourcegraph.com/sourcegraph/srclib-python
   - go get -d -v ./... && go build -v ./...
-  - make test-dependencies
```

Comment:
Toolchain should now install dependencies automatically. Like if you depends on `requests` and it's either in *requirements*.txt or setup.py, it will automatically install them in the SourceUnits virtualenv.